We do not want to check coverage for this project.

--- hatch-requirements-txt-0.4.1/tests/requirements.txt.orig
+++ hatch-requirements-txt-0.4.1/tests/requirements.txt
@@ -1,10 +1,8 @@
 coincidence>=0.2.0
-coverage>=5.1
 dist-meta>=0.5.0
 domdf-python-tools>=3.3.0
 importlib-metadata>=3.6.0
 pkginfo>=1.8.3
 pytest>=6.0.0
-pytest-cov>=2.8.1
 pytest-randomly>=3.7.0
 pytest-timeout>=1.4.2
--- hatch-requirements-txt-0.4.1/tox.ini.orig
+++ hatch-requirements-txt-0.4.1/tox.ini
@@ -67,7 +67,7 @@
 deps = -r{toxinidir}/tests/requirements.txt
 commands =
     python --version
-    python -m pytest --cov=hatch_requirements_txt -r aR tests/ {posargs}
+    python -m pytest -r aR tests/ {posargs}
 
 [testenv:.package]
 setenv =
